model/fv_control.F90

```diff
@@ -578,7 +581,11 @@ subroutine fv_control_init(Atm, dt_atmos, this_grid, grids_on_this_pe, p_split,
         call read_namelist_molecular_diffusion_nml(Atm(this_grid)%nml_filename, &
                   Atm(this_grid)%flagstruct%ncnst,  Atm(this_grid)%flagstruct%nwat)
      endif
+
+     call read_namelist_fv_processmodel(Atm(this_grid)%nml_filename)
+
```

Comment:
In fv_restart.F90, you protected similar logic for module use functions via statement with a compile-time conditional.  Should you do the same here?  Are you advocating the process model code always be compiled, or conditionally compiled?

Reply:
Yes, this should also have the GFS_PHYS flag.